site/.vitepress/config.ts

```diff
@@ -133,19 +133,19 @@ export default defineConfig({
           },
           { text: "How to Batch Transactions", link: "/batching-transactions" },
```

Comment:
do you want to lower case this sentence too if we're moving away from capital-case?

Reply:
I actually think we should keep al these capital case, just as the other sections of the sidebar are capital case. can you revisit this?

Reply:
yes, revisited and kept uppercase for titles everywhere!

Reply:
These titles are still lower case.